src/kameleon/util/search.clj

```diff
@@ -2,8 +2,8 @@
   (:require [clojure.string :as str]))
 
 (defn format-query-wildcards
-  [search-term]
-  (str/replace search-term #"[%_*?]" {"%" "\\%",
+  [search-term] 
```

Comment:
It's tiny, but please delete this extra space as well. We don't have any sort of autoformatting in place for Clojure, so we keep that stuff up by hand.